### ISSUE TITLE
A Property with default is being set even though it has required flag

### ISSUE THREAD

**Issue by ANISH-GOTTAPU** [open-traffic-generator/openapiart#57]
A Property with default is being set even though it has required flag, I think we can limit this with two options
1) update model guide to `either include default or required`.
2) Throw an exception on the client if a property has both `required and default`.

Ex: https://github.com/open-traffic-generator/openapiart/blob/main/openapiart/tests/config/config.yaml#L91
```
    EObject:
      x-include:
      - '../common/common.yaml#/components/schemas/GlobalObject'
      required: [e_a, e_b]
      properties:
        e_a:
          type: number
          format: float
          default: 1.0
        e_b:
          type: number
          format: double
          default: 2.0```

**Comment by ajbalogh** [comment]
The modeling guide should be updated to state it can only be one or the other and not both.
The bundler.py should be enhanced to enforce that behavior.

**Comment by Rangababu-R** [comment]
available in 0.1.31
